packages/map-template/src/components/Search/Search.scss

```diff
@@ -47,4 +47,8 @@
         padding: var(--spacing-medium) var(--spacing-x-small) var(--spacing-medium) var(--spacing-x-small);
     }
 
+    span {
```

Comment:
Consider targeting using a BEM element selector (`&__label`? 🤔 ), just to avoid problems down the line if we add a `<span>` in another place in the component.

Reply:
Good point ✅